superset/assets/visualizations/markup.js

```diff
@@ -23,9 +24,11 @@ function markupWidget(slice, payload) {
     <iframe id="${iframeId}"
       frameborder="0"
       height="${slice.height()}"
-      sandbox="allow-scripts">
+      sandbox="allow-same-origin allow-scripts allow-top-navigation allow-popups">
```

Comment:
Do we really need the ```allow-top-navigation allow-popups``` part?

Reply:
These are to fix #3672. When the markup slice was moved to use iframe, any link inside the slice would only open within the slice and links meant to be opened in another tab/window did nothing.

We use this for documentation and referencing other sites. 
- `allow-top-navigation` lets users create a link with `target="_top"` that will open the link in the same tab
- `allow-popups` lets users create a link with `target="_blank"` that will open the link in a new tab

Without these two fields links will only open inside the iframe itself